src/Executor/ReferenceExecutor.php

```diff
@@ -116,7 +116,7 @@ public function doExecute() : Promise
      * execute, which we will pass throughout the other execution methods.
      *
      * @param mixed[]             $rootValue
-     * @param mixed[]             $contextValue
+     * @param mixed               $contextValue
```

Comment:
`$rootValue` here should also be typed as `mixed` then

Reply:
done